developer-docs/bolt-server.md

```diff
@@ -229,52 +224,62 @@ curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
 ### With the ruby client
 
 There is a simple ruby client that can be used to make requests to a local
-bolt_server during development at `scripts/server_client.rb`. This server
+bolt server during development at `scripts/server_client.rb`. This server
 expects to use the puppet-server container and target nodes from bolts spec
-environment so run `docker-compose up -d --build` from the spec directory before
-trying to use it.
+environment so run `docker-compose up -d --build` from the spec directory
+before trying to use it.
 
 ```
-bundle exec scripts/server_client.rb sample::echo ssh '{"message": "hey"}'
+bundle exec scripts/server_client.rb sample::echo <TARGET> '{"message": "hey"}'
+```
+Where `<TARGET>` is either:
+* A vmpooler VM. To use this, replace `<TARGET>` above with the hostname.
+* One of the containers brought up by the `docker-compose` in the `spec` directory. To use these, you'll want to get the IP of the container bolt-server is running on:
```

Comment:
I think it is worth pointing out that any task in `bolt/spec/fixtures/modules/sample` will exist in the `puppetserver` image.  For example you could run the task defined with `init.sh` with `bundle exec scripts/server_client.rb sample bolt:bolt@172.18.0.1:20022  '{"message": "hey"}' ` 
with expected output:
```
{
  "node": "172.18.0.1",
  "status": "success",
  "result": {
    "_output": "hey\n"
  }
}
```

Reply:
`bolt/spec/fixtures/modules`

Reply:
Right. I guess we could just link to https://github.com/puppetlabs/bolt/blob/d0473f31ba9fefdf715e7ab7b76aeef05ab91f3d/spec/docker-compose.yml#L29